core/templates/dev/head/domain/exploration/InteractionObjectFactory.js

```diff
@@ -51,7 +52,7 @@ oppia.factory('InteractionObjectFactory', [
           return hint.toBackendDict();
         }),
         id: this.id,
-        solution: {}
```

Comment:
Oh no. Does this mean we already have {} in the backend storage? If so, do we need another exploration migration to make everything consistent?